app/models/ncr/work_order.rb

```diff
@@ -197,6 +197,10 @@ def restart_budget_approvals
       proposal.root_step.initialize!
     end
 
+    def self.expense_type_options
+      EXPENSE_TYPES.map { |et| [et, et] }
```

Comment:
`expense_type` instead of `et`? longer, but I spent a few seconds thinking about :alien: when I first saw this method